CSS/style.css

```diff
@@ -22,10 +24,80 @@ html {
   scroll-behavior: smooth;
 }
 
+/* css reset */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+iframe {
+  border-radius: 10px;
+}
+
 body {
+  font-family: "Rubik", sans-serif;
   width: 100%;
   max-width: 1024px;
   min-height: 100vh;
   height: max-content;
-  font-family: "Rubik", sans-serif;
+  margin: 0 auto;
+  font-size: 62.5%;
+}
+
+h1 {
+  font-size: 4.8rem;
+  line-height: 1.05em;
+  font-weight: 800;
+}
+
+h2 {
+  font-size: 3.6rem;
+  line-height: 1.25em;
+  font-weight: 700;
+}
+
+h3 {
+  font-size: 2.8rem;
+  line-height: 1.25em;
+  font-weight: 600;
+}
+
+h4 {
+  font-size: 1.8rem;
+  line-height: 1.22em;
+  font-weight: 500;
+}
+
+p {
+  font-size: 1rem;
+  line-height: calc(1.5em + 0.2vw);
+}
+
+main {
+  width: 100%;
+  padding: 0 1rem;
+  margin: 0 auto;
+  min-height: 100vh;
+}
+
+.heading {
+  text-align: center;
+}
+
+.sub-heading {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.heading-button-container {
+  display: flex;
+  margin: 25px auto;
+  gap: 30px;
+  justify-content: center;
+}
+
+.heading-button-container > a {
+  font-size: 20px;
+  text-decoration: none;
+  padding: 15px 50px 15px 50px;
```

Comment:
You can consider using padding shorthand for this. I think it can be re-written as:

padding: 15px 50px

Reply:
fixes